examples/transform_rotation.rs

```diff
@@ -96,3 +115,33 @@ fn setup(mut commands: Commands) {
         }
     }
 }
+
+fn options_panel(mut egui_context: ResMut<EguiContext>, mut options: ResMut<Options>) {
```

Comment:
Can't this be done with `bevy-inspector-egui` instead of adding an extra dependency?

Reply:
Didn't know you could do that, pretty cool!